src/domains/ecommerce-common/03_storage.tf

```diff
@@ -43,6 +43,29 @@ resource "azurerm_private_endpoint" "storage_private_endpoint" {
   tags = var.tags
 }
 
+
+resource "azurerm_private_endpoint" "storage_transient_private_endpoint" {
+  count = var.env_short != "d" ? 1 : 0
+
+  name                = "${local.project}-tr-storage-private-endpoint"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.storage_ecommerce_rg.name
+  subnet_id           = module.ecommerce_storage_snet.id
+  private_dns_zone_group {
+    name                 = "${local.project}-storage-private-dns-zone-group"
+    private_dns_zone_ids = [data.azurerm_private_dns_zone.storage.id]
+  }
+
+  private_service_connection {
```

Comment:
pay attention on this configuration, i think it's ok but maybe better to investigate if we can use the same private link defined for old queue
https://learn.microsoft.com/en-us/azure/private-link/private-endpoint-overview